mitodl/micromasters
# Added MIT Micromasters graphic and updated title

#### What are the relevant tickets?
Fixes #95 

#### What's this PR do?
Uses the new MIT Micromasters graphic, and updates the title from "Micromasters" to "MIT Micromasters"

#### Where should the reviewer start?

#### How should this be manually tested?
Open the dashboard and verify that you see a graphic with 'Micromasters' right next to MIT.

#### Any background context you want to provide?

#### Screenshots (if appropriate)

#### What GIF best describes this PR or how it makes you feel?
